news/239.bugfix

```diff
@@ -0,0 +1 @@
+Prevent breaking changes inside a minor Plone release
```

Comment:
Can you be more specific? Something like:
Restore selectedTabs code that was removed in 3.4.1 to prevent breaking changes inside a minor Plone release.

Otherwise the PR is good for me.

Reply:
Done! Thanks for the suggestion :)